### ISSUE TITLE
Failed Setup

### ISSUE THREAD

**Issue by jjhplus9** [uvjim/linksys_velop#447]
Integration was working fine.
Now it gives the following error:

Logger: custom_components.linksys_velop
Source: custom_components/linksys_velop/__init__.py:326
Integration: Linksys Velop (documentation, issues)
First occurred: 12:48:11 (8 occurrences)
Last logged: 12:52:30

_async_get_mesh_data --> 'NoneType' object has no attribute 'startswith'
![image](https://github.com/uvjim/linksys_velop/assets/55134856/a0a77406-af68-4d3d-aadc-7a96f0e6fad7)
[home-assistant_linksys_velop_2024-01-29T12-58-00.279Z.log](https://github.com/uvjim/linksys_velop/files/14084496/home-assistant_linksys_velop_2024-01-29T12-58-00.279Z.log)

**Comment by uvjim** [comment]
Has anything else changed in the environment? Anything else shown in the logs? Could you turn debugging on and see what is reported in the logs?

**Comment by jjhplus9** [comment]
I attached logs to the OP using the debug capture.
There are unrelated errors in the logs but nothing else ref the velop integration

**Comment by uvjim** [comment]
Sorry. The email just kept opening the image rather than logs. I'll take a look and see if I can work out what's going on.

Nothing changed in your environment though? I'm not having the issue here, but I'm sure mine is just a simple setup compared to most.

**Comment by jjhplus9** [comment]
Nothing relevant comes to mind.
Although am building out my HA I don't know of a change recently.
I did have the issue where the velop nodes would go "away" from HA every fifteen or twenty minutes but I learned to live with that.
The complete lack of nodes is new

**Comment by uvjim** [comment]
This is concerning as responses go from the API.  Does the Linksys app/web UI respond correctly?

```
2024-01-29 12:57:58.321 DEBUG (MainThread) [pyvelop.jnap.verbose] Request.execute --> action: http://linksys.com/jnap/nodes/networkconnections/GetNodesWirelessNetworkConnections --> payload: {} --> response: {'result': '_ErrorUnexpected', 'error': '/usr/local/lib/lua/5.1/nodes/util/client.lua:66: DevDb_GetDeviceByMacAddr mac = e0:4b:41:00:65:2f failed. Error code = 24. Error message = Server responded with fatal error: Cannot execute sqlite command: database disk image is malformed @ SqlHelper::read_as_value @ ClientConn::process_data.'}
```

**Comment by jjhplus9** [comment]
I'll go check now

**Comment by jjhplus9** [comment]
![Screenshot 2024-01-30 at 00 05 59](https://github.com/uvjim/linksys_velop/assets/55134856/2cdc1d91-be8a-4cb2-a2d2-74d3db195aab)
OK, so although the ios APP has access, the web UI doesnt, which I am sure must be related to the error I am seeing.
I'm going to raise a ticket with Linksys

I'll update here as and when I get a response

**Comment by uvjim** [comment]
@jjhplus9 - did you have any luck with Linksys?

**Comment by jjhplus9** [comment]
@uvjim No, made me do a 'hard reset' which of course did absolutely nothing.
seems to be a certificate issue, but they where zero help
last recommendation was to buy a new master node from a newest generation, which I am unlikely to do.
They claimed it would be compatible with the OG velop nodes, but I have found no evidence that that would be so.

**Comment by uvjim** [comment]
@jjhplus9 - interesting. I thought that typically the newer version would have to be the primary node. It would work with the older secondary nodes mind you. However, not true of the newest of the new. My understanding is that they use a slightly different tech and could not work with the older versions.

**Comment by jjhplus9** [comment]
I asked them if I bought a velop 6e master (MX6201) would it work with the old nodes (I have OG gen one nodes) - support said yes - I asked them to produce some evidence that this would work and they could not. Conversely, I saw plenty (or at least some) reports that older nodes do not work with 6e master.
Although I have lost monitoring from HA, the rest works fine for my families purpose.
Don't really want to drop £150+ on a new node with all the setup time of a new master only for it to not work.
Plus given the crippled UI I doubt I could re-add the nodes to the old master again so I would be snookered and would have to buy five new nodes and that's help expensive.
So I am minded to leave well enough alone.
But super open to trying anything else anyone can suggest.

**Comment by uvjim** [comment]
It can seemingly be done...

https://www.linksys.com/support-article?articleNum=50933

**Comment by jjhplus9** [comment]
I do think that page existed when I looked or at least I didn't find it.
It may be worth a try given some evidence that it could work.
Amazon having a 30 day returns policy and all that.
Are there any other reports of the issue?

**Comment by uvjim** [comment]
> Are there any other reports of the issue?

I've not seen any. Doesn't mean it isn't happening though. None of my nodes seem to be exhibiting this and some of them are the really old little nodes. I even have the little plug-in node on the mesh.

**Comment by uvjim** [comment]
Whilst they mention the MBE7000 series (£400 for one unit 😱) it doesn't say that it'll only work with that series.

**Comment by jjhplus9** [comment]
yeah, MBE7000 is not happening here...

**Comment by uvjim** [comment]
I'll close this off, but gimme a shout if you do get to use the integration again and anything weird happens.

**Comment by jjhplus9** [comment]
Will do - and thank you
